cmd/tools/migration/meta/210_to_220.go

```diff
@@ -142,21 +144,36 @@ func combineToSegmentIndexesMeta220(segmentIndexes SegmentIndexesMeta210, indexB
 			if !ok {
 				return nil, fmt.Errorf("index build meta not found, segment id: %d, index id: %d, index build id: %d", segID, indexID, record.GetBuildID())
 			}
+
+			fileIDs := make([]int64, len(buildMeta.GetIndexFilePaths()))
+			for i, filePath := range buildMeta.GetIndexFilePaths() {
+				parts := strings.Split(filePath, "/")
+				if len(parts) != 0 {
+					return nil, fmt.Errorf("invaild index file path: %s", filePath)
+				}
+
+				fileID, err := strconv.ParseInt(parts[len(filePath)-1], 10, 64)
```

Comment:
What's the meaning of `fileID` here? The old format of index file path is: `index_files/{index build id}/{version}/{partition id}/{segment id}/{key}`, where key is given by `knowhere`, for example `IVF-1`, `IVF-2`, `indexParams` and etc. So `parts[len(filePath)-1]` won't be parsed to `fileID`.